Description: set prefix to /usr
 In a bid to keep debian/rules simpler, this patch changes PREFIX to /usr
 .
 vimb (3.6.0-1) unstable; urgency=medium
 .
   * Initial release (Closes: #982010)
Author: Martin Atukunda <matlads@gmail.com>
Bug-Debian: https://bugs.debian.org/982010

--- vimb-3.6.0.orig/config.mk
+++ vimb-3.6.0/config.mk
@@ -2,7 +2,7 @@ ifneq ($(V),1)
 Q := @
 endif
 
-PREFIX           ?= /usr/local
+PREFIX           ?= /usr
 BINPREFIX        := $(DESTDIR)$(PREFIX)/bin
 MANPREFIX        := $(DESTDIR)$(PREFIX)/share/man
 EXAMPLEPREFIX    := $(DESTDIR)$(PREFIX)/share/vimb/example
